### PR TITLE
MOVE-1130: Add search bar to Admin Console

### PR DESCRIPTION
**Pull request by imadsyed333**
# Issue Addressed
This PR closes MOVE-1130

# Description
Added search functionality to the User Permissions page in the Admin Console. Works in tandem with current implementation of pagination.

image

# Tests
Tested in MOVE local v1.11.1 in Chrome, Firefox, and Edge

### CODE REVIEW

**Review by spencermountain — APPROVED** [review]
👑

File: lib/controller/UserController.js [hunk 1]
```diff
@@ -33,12 +33,13 @@ UserController.push({
...
         limit: Joi.number().integer().min(0).required(),
         offset: Joi.number().integer().min(0).required(),
+        search: Joi.string().allow(''),
       },
     },
   },
   handler: async (request) => {
-    const { limit: pageLimit, offset: pageOffset } = request.query;
-    const users = await UserDAO.getUsersPagination(pageLimit, pageOffset);
+    const { limit, offset, search } = request.query;
```
Comment (ryanpdaley) [inline lib/controller/UserController.js:41]: This still works, right? 😄 I usually only rename destructured variables when I have to... haha

**Review by ryanpdaley — APPROVED** [review]
Assuming the one comment is cool! 👍

File: lib/controller/UserController.js (hunk 1 above)
Comment (imadsyed333) [reply lib/controller/UserController.js:41]: Yeah this should still work. I didn't know at the time that we could simple use the variable names provided when de-structuring 😅

**Review by lowsound42 — APPROVED** [review]
💯

**Pull request merged**